features/docs/gegeven-stap-definities-expressief.feature

```diff
@@ -0,0 +1,146 @@
+      |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                                                           | 9999,0,0,P,000000012,P2                               |
+      |      | partner      | INSERT INTO public.lo3_pl_persoon(pl_id,volg_nr,burger_service_nr,geslachts_naam,relatie_start_datum,relatie_start_plaats,relatie_start_land_code,persoon_type,stapel_nr) VALUES($1,$2,$3,$4,$5,$6,$7,$8,$9) | 9999,0,000000024,P3,gisteren - 20 jaar,0518,6030,R,0  |
+      |      | kind-1       | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr) VALUES($1,$2,$3,$4,$5)                                                                                             | 9999,0,0,K,000000036                                  |
+      | 2    | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING *                                                        | 0                                                     |
+      |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                                                           | 10000,0,0,P,000000024,P3                              |
+      |      | partner      | INSERT INTO public.lo3_pl_persoon(pl_id,volg_nr,burger_service_nr,geslachts_naam,relatie_start_datum,relatie_start_plaats,relatie_start_land_code,persoon_type,stapel_nr) VALUES($1,$2,$3,$4,$5,$6,$7,$8,$9) | 10000,0,000000012,P2,gisteren - 20 jaar,0518,6030,R,0 |
+      |      | kind-1       | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr) VALUES($1,$2,$3,$4,$5)                                                                                             | 10000,0,0,K,000000036                                 |
+      | 3    | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING *                                                        | 0                                                     |
+      |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                                                           | 10001,0,0,P,000000036,P1                              |
+      |      | ouder-1-1    | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                                                           | 10001,0,0,1,000000012,P2                              |
```

Comment:
bij deze ouderrelaties (persoon_type='1' of '2') moet datum familierechtelijke betrekking worden toegevoegd. Die heeft als standaardwaarde de geboortedatum van de persoon (gisteren - 17 jaar)